packaging/torcharrow/meta.yaml

```diff
@@ -24,16 +24,15 @@ requirements:
     - cffi
     - glog==0.4.0
     - libsodium
-    {{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
+    - pytorch
```

Comment:
curious: why do we need to build with pytorch? (since we don't build text UDFs by default)

Reply:
this is in the run section, which specifies the dependencies needed to use torcharrow. I think torcharrow still imports pytorch right?